packages/core/src/App/Constants/app-config.js

```diff
@@ -1,3 +1,4 @@
 export const website_domain = 'Deriv.app';
 export const website_name = 'Deriv';
 export const default_title = website_name;
+export const cookie_expires = 7;
```

Comment:
perhaps we could rename this to `cookie_banner_expires_in_days` or something similar as well

Reply:
Bump.